.styleci.yml

```diff
@@ -2,4 +2,10 @@ preset: psr2
 
 enabled:
   - long_array_syntax
-  - duplicate_semicolon
\ No newline at end of file
+  - duplicate_semicolon
+
+finder:
+  not-name:
+    # exclude PHP7 illegal annotation classes to avoid false alert. those must not be attempted to be used in PHP 7
+    - lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Float.php
+    - lib/Doctrine/ODM/PHPCR/Mapping/Annotations/String.php
```

Comment:
`Float.php` and `String.php` should be fine here, since you really don't want to load any classes with those names, where they're located, right?